server/services/odata.service.js

```diff
@@ -28,12 +28,33 @@ const PREFER = 'Prefer'
 const ODATA_VERSION_NUMBER = '4.0'
 const PREFER_REPRESENTATION = 'return=representation'
 
+const _replaceUnsafeCharacters = certificateNumber => {
+  return certificateNumber
+    ? certificateNumber
+        .replaceAll('%', '%25')
+        .replaceAll('+', '%2B')
+        .replaceAll('&', '%26')
+        .replaceAll('#', '%23')
+        .replaceAll('|', '')
+        .replaceAll('<', '')
+        .replaceAll('>', '')
+        .replaceAll('^', '')
+        .replaceAll('\\', '')
+        .replaceAll('{', '')
+        .replaceAll('}', '')
+        .replaceAll('[', '')
+        .replaceAll(']', '')
+        .replaceAll('\'', '\'\'')
```

Comment:
I am using VS Code with standard JS and I have a "format on save" option which replaces the single quotes on line 47 to this:

.replaceAll("'", "''")

i.e. uses double quotes instead of escaping the single quotes. Can you check your config to see if you have the same? Will be better if we are auto-formatting the code the same way.